test/hotspot/jtreg/compiler/loopopts/TestCountedLoopCastIV.java

```diff
@@ -171,6 +183,10 @@ private void runTest4() {
     }
 
     public static void main(String[] args) {
-        TestFramework.runWithFlags("-XX:LoopUnrollLimit=0");
+        if (args != null && args.length > 0 && args[0].equals("DisableUnroll")) {
+            TestFramework.runWithFlags("-XX:LoopUnrollLimit=0");
+        } else {
+            TestFramework.run();
+        }
```

Comment:
I would recommend checking that there is no "unexpected" input here.
```suggestion
        if (args != null && args.length > 0 && args[0].equals("DisableUnroll")) {
            TestFramework.runWithFlags("-XX:LoopUnrollLimit=0");
        } else {
            if (args.length != 0) { throw new RuntimeException("Unexpected args"); }
            TestFramework.run();
        }
```